guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>(1) &nbsp;FullyConnectedNetwork and is imported from <s>arcgsi.learn</s> <strong>arcgis.learn</strong> followed by initialization&nbsp;</p><p>(2) &nbsp;fields are transformed <s>to</s> <strong>into </strong>numerical type by one-hot encoding or embedding</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>